runtime/vm/createramclass.cpp

```diff
@@ -2857,7 +2857,7 @@ internalCreateRAMClassFromROMClassImpl(J9VMThread *vmThread, J9ClassLoader *clas
 						J9ARRAYCLASS_SET_STRIDE(ramClass, J9_VALUETYPE_FLATTENED_SIZE(elementClass));
 					}
 				} else {
-					J9ARRAYCLASS_SET_STRIDE(ramClass, (1 << (((J9ROMArrayClass*)romClass)->arrayShape & 0x0000FFFF)));
+					J9ARRAYCLASS_SET_STRIDE(ramClass, ((U_64) 1 << (((J9ROMArrayClass*)romClass)->arrayShape & 0x0000FFFF)));
```

Comment:
`(U_64) 1` -> `((U_64) 1)`

The cast needs the extra brackets

Reply:
Fixed